pkg/harvester/edit/harvesterhci.io.setting.vue

```diff
@@ -58,7 +59,13 @@ export default {
 
     return {
       setting,
-      description:        isHarvester ? t(`advancedSettings.descriptions.harv-${ this.value.id }`) : t(`advancedSettings.descriptions.${ this.value.id }`),
+      description: isHarvester ? t(
+        `advancedSettings.descriptions.harv-${ this.value.id }`,
+        this.value.id === 'upgrade-config' ? { version: this.getUpgradeDocVersion() } : {}
+      ) : t(
+        `advancedSettings.descriptions.${ this.value.id }`,
+        this.value.id === 'upgrade-config' ? { version: this.getUpgradeDocVersion() } : {}
```

Comment:
Can we make it more generic in case other settings need similar behavior in the future? e.g. 
```js
[HCI_SETTING.UPGRADE_CONFIG]:                         {
    kind:         'json',
    from:         'import',
    ...
    docLink
  },
  ```